### PR TITLE
add Temurin 19 linux installers

### PR DESCRIPTION
**Pull request by gdams**
Draft until Linux s390x binaries are available

### CODE REVIEW

File: linux/jdk/debian/src/main/packaging/temurin/19/debian/rules [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/make -f
+
+pkg_name = temurin-19-jdk
+priority = 1811
+jvm_tools = jar jarsigner java javac javadoc javap jcmd jconsole jdb jdeprscan jdeps jfr jhsdb jimage jinfo jlink jmap jmod jpackage jps jrunscript jshell jstack jstat jstatd jwebserver keytool rmiregistry serialver jexec jspawnhelper
+amd64_tarball_url = https://github.com/adoptium/temurin19-binaries/releases/download/jdk-19%2B36/OpenJDK18U-jdk_x64_linux_hotspot_19_36.tar.gz
```
Comment (zdtsw) [inline linux/jdk/debian/src/main/packaging/temurin/19/debian/rules:6]: these links need to be updated  OpenJDK19U-

Comment (gdams) [reply linux/jdk/debian/src/main/packaging/temurin/19/debian/rules]: updated

**Review by karianna — APPROVED** [review]
LGTM

**Comment by karianna** [comment]
@gdams RH and Suse failing?

**Comment by zdtsw** [comment]
> @gdams RH and Suse failing?

dont think OpenJDK19U-jdk_s390x_linux_hotspot_19_36.tar.gz is ready to be download ?

**Review by tellison — APPROVED** [review]

**Pull request merged**